rotkehlchen/utils/misc.py

```diff
@@ -384,3 +384,29 @@ def create_order_by_rules_list(
     order_by_attribute = data['order_by_attributes'] if data['order_by_attributes'] is not None else [timestamp_field]  # noqa: E501
     ascending = data['ascending'] if data['ascending'] is not None else [False]
     return list(zip(order_by_attribute, ascending))
+
+
+def levenshtein_distance(s1: str, s2: str) -> float:
```

Comment:
Hmmm this does not look like an efficient levensthein implementation. From your link probably the last one is the most performant.

Have you tried using https://github.com/fujimotos/polyleven and timing it against this implementation to see the difference?

Also this can't return a float. It returns an int.